Lib/pdb.py

```diff
@@ -555,7 +561,7 @@ def bp_commands(self, frame):
         Returns True if the normal interaction function must be called,
         False otherwise."""
         # self.currentbp is set in bdb in Bdb.break_here if a breakpoint was hit
-        if getattr(self, "currentbp", False) and \
+        if self.currentbp and \
```

Comment:
```suggestion
        if getattr(self, "currentbp", False) and \
```

Add back the getattr from removing currentbp